src/arch/src/aarch64/regs.rs

```diff
@@ -123,7 +127,8 @@ macro_rules! arm64_core_reg_id {
 // https://elixir.bootlin.com/linux/v4.20.17/source/arch/arm64/include/uapi/asm/kvm.h#L203
 macro_rules! arm64_sys_reg {
     ($name: tt, $op0: tt, $op1: tt, $crn: tt, $crm: tt, $op2: tt) => {
-        const $name: u64 = KVM_REG_ARM64 as u64
+        /// System register constant
+        pub const $name: u64 = KVM_REG_ARM64 as u64
```

Comment:
this does not need to be public

Reply:
I have made this public so that I can use it inside the Integration Tests, for the purpose of finding the MIDR_EL1. Else, it says it is a private constant. I was also thinking of creating a global variable out of it and export that, but I am not sure about this approach. What do you think?